tests/unit/completion/test_models.py

```diff
@@ -50,8 +51,12 @@ def _get_completions(model):
             desc = category.child(j, 1)
             misc = category.child(j, 2)
             entries.append((name.text(), desc.text(), misc.text()))
-        completions.append((category.text(), entries))
-    return completions
+        actual[category.text()] = entries
+    for cat_name, expected_entries in expected.items():
+        assert cat_name in actual
+        actual_items = actual[cat_name]
+        for expected_item in expected_entries:
+            assert any(item for item in actual_items if item == expected_item)
```

Comment:
I *think* this does the right thing because `any([])` is `False`, and `item` always would be truthy (as it's a non-empty list).

But wouldn't `assert expected_item in actual_items` do the same?